### DISCUSSION TITLE
Ein Kuchen mmmmhhh

### DISCUSSION THREAD

**Discussion by RLeo** [Bootstrap-Academy/Bootstrap-Academy#91] [Ideas]
Ich hätte gerne einen Kuchen im Profil. 
Ein Skill-Kuchen(-Diagramm) bei dem die Root-Skills je nach XP-Anteil angeordnet sind.
Ist eher interessant wenn der Profil öffentlich wird. Dann sagt der Kuchen was über die Lern-Prioritäten des Users aus.
Gleichzeitig kann man auch ein Vue, TypeScript oder CSS-Diagramm-Video machen in dem man mit diesem Beispiel beibringt wie man mit ein Diagramm bastelt.
Was haltet ihr von der Spielerei?

**Comment by Cameo007** [comment]
Gute Idee. Vor allem das man das mit einem Tut verbindet.

**Comment by TheMorpheus407** [comment]
Sehr spannende Idee, danke! Du kannst gerne daraus ein Feature Request machen
